schema-registry/src/main/resources/confluent.properties.orig

```diff
@@ -0,0 +1,3 @@
+# This is a placeholder for storing the properties need to connect to Confluent Cloud
+# Create a new properties file by copying this file to one named confluent.properties
+# The confluent.properties file is ignored by Git - never checkin your credentials!
```

Comment:
add some placeholders below here with the expected fields and placeholders for credential values?

Reply:
Fair enought